scss/_base_button.scss

```diff
@@ -98,6 +98,21 @@
         margin-right: -$sph--small;
       }
     }
+
+    &.is-small {
+      & [class*='p-icon'] {
+        margin-left: calc($sph--small / 2);
```

Comment:
I think this value is a fair assumption, but we have a variable for that, it's `$sph--x-small`. Could you use it instead of dividing by 2?

Reply:
Oops, I missed that variable earlier. It's fixed now!